source/cppfront.cpp

```diff
@@ -1805,6 +1734,25 @@ class cppfront
         auto last_was_prefixed = false;
         auto saw_dollar        = false;
 
+        auto args = std::optional<std::vector<text_with_pos>>{};
```

Comment:
Here, store two more values (we could bundle these three in a struct but I'm fine with three similarly named locals):

```
        auto args_open_pos  = source_position{};
        auto args_close_pos = source_position{};
```

Reply:
That was also one of my ideas that I will try.